### PR TITLE
Nikita/withdraw update

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?
1.  Update Withdraw page component with MUI.
2. Update Deposit page with MUI component.
3. Mobile responsive of `WithdrawPage` and `DepositPage` 

## Context
- Figma 
https://www.figma.com/file/Aa3IfDUO68aCNr8nZTuQ8S/Product-Design-with-MUI?node-id=106%3A17356
https://www.figma.com/file/Aa3IfDUO68aCNr8nZTuQ8S/Product-Design-with-MUI?node-id=106%3A16216

### CODE REVIEW

**Comment by hammeiam** [comment]
Please add some details about what we should be looking for in this PR, as well as screenshots if relevant. Otherwise, please use a `Draft PR` until you're ready for review. 

Looks like there's a possible wrapping issue with the token swap input
Screen Shot 2022-02-21 at 8 02 35 PM

**Comment by nkrulikovsky** [comment]
Yes, It is still having somethings to update.

File: src/components/WithdrawPage.tsx [hunk 1]
```diff
@@ -159,108 +167,101 @@ const WithdrawPage = (props: Props): ReactElement => {
...
-                      <span className="slippage">{t("priceImpact")}</span>
-                    )}
-                    <span
-                      className={
-                        "value " +
-                        (reviewData.priceImpact.gte(0) ? "bonus" : "slippage")
+              <Stack spacing={3}>
+                {tokensData.map((token, index) => (
+                  <div key={index}>
+                    <TokenInput
```
Comment (ug02fast) [inline src/components/WithdrawPage.tsx:173]: can `key=` attribute go directly to `TokenInput` component as a prop and remove the div?

File: src/components/WithdrawPage.tsx [hunk 2]
```diff
@@ -159,108 +167,101 @@ const WithdrawPage = (props: Props): ReactElement => {
...
+                      {...token}
+                      // inputValue={parseFloat(token.inputValue).toFixed(5)}
```
Comment (ug02fast) [inline src/components/WithdrawPage.tsx:175]: If this isn't used and no comments around it then del. @hammeiam any objections?

**Review by ug02fast — COMMENTED** [review]
questions

File: src/components/WithdrawPage.tsx [hunk 3]
```diff
@@ -82,6 +84,8 @@ const WithdrawPage = (props: Props): ReactElement => {
 
   const { gasPriceSelected } = useSelector((state: AppState) => state.user)
   const [currentModal, setCurrentModal] = useState<string | null>(null)
+  const theme = useTheme()
+  theme
```
[the remark this answers is not in the document]
Comment (nkrulikovsky) [reply src/components/WithdrawPage.tsx:88]: Oh, it should be removed.

File: src/components/DepositPage.tsx [hunk 4]
```diff
@@ -217,24 +217,26 @@ const DepositPage = (props: Props): ReactElement => {
...
-              }}
-              className="divider"
-            ></div>{" "}
-            <PoolInfoCard data={poolData} />
-          </div>
-        </div>
+          <Paper>
+            <Box p={4}>
+              <MyShareCard data={myShareData} />
+              <div
```
[the remark this answers is not in the document]
Comment (nkrulikovsky) [reply src/components/DepositPage.tsx]: Yes, we can. Fixed!

**Review by ug02fast — APPROVED** [review]

**Review by hammeiam — APPROVED** [review]
This looks great! This will be a huge usability increase for a lot of our users

**Pull request merged**